docs/crocks/Pair.md

```diff
@@ -0,0 +1,77 @@
+# Pair
+
+`Pair c a`
+
+--
+
+--
+
+```js
+--
+```
+
+`Pair` exposes the following functions on the constructor and instance:
+
+| Constructor | Instance |
+|:---|:---|
+| [`type`](#type) | [`ap`](#ap), [`bimap`](#bimap), [`chain`](#chain), [`concat`](#concat), [`equals`](#equals), [`fst`](#fst), [`inspect`](#inspect),  [`map`](#map), [`merge`](#merge), [`of`](#of), [`type`](#type), [`snd`](#snd), [`swap`](#swap), [`value`](#value) |
+
+## Constructor
+
+### type
+
+`() -> String`
+
+## Instance
+
+### ap
+
+`Pair m, Semigroup < c, a > => m c a ~> m c a -> m c a`
+
+### bimap
+
+`Pair m => m c a ~> (c -> d) -> (a -> b) -> m d b`
+
+### chain
+
+`Pair m => m c (a -> a) ~> (a -> m c a) -> m c a`
```

Comment:
don't know what this is about. 
why is there a function `(a -> a)`?
there needs to be a semigroup constraint on c.
That function makes no sense.